docs/source/usage/plugins/xrayDiffraction.rst

```diff
@@ -0,0 +1,103 @@
+.. _usage-plugins-xrayDiffraction:
+
+X-ray diffraction
+-----------------
+
+This plugin calculates X-ray diffraction from particle positions.
+
+.. math::
+
+   I({\bf q}) &= \frac{F({\bf q})F({\bf q})^*}{Np} \\
+   F({\bf q}) &= \sum^{N}_{j=1} f_j w_j \exp(i {\bf q \cdot \bf r_j})
```

Comment:
@ejcjason as far as I can see in [your paper](http://journals.iucr.org/s/issues/2018/02/00/co5094/co5094.pdf) equation (5) also has a 2 * pi multiplier under the exponent, but the code does not have it, and the docs correspond to the code. What is the reason for it: is the multiplier included in q already?

Reply:
@ejcjason Could you please have a look to this question!

Reply:
@ejcjason PING!